### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
From what I can see, it seems that you've used the old version of the NestJS logo :( 

Current version: https://nestjs.com/img/logo-small.svg

That's very likely something I should have cleanup in the past (all the PNG versions of the favicon in the repo are still using the old logo).

**Comment by Arhell** [comment]
> From what I can see, it seems that you've used the old version of the NestJS logo :(
> 
> Current version: https://nestjs.com/img/logo-small.svg
> 
> That's very likely something I should have cleanup in the past (all the PNG versions of the favicon in the repo are still using the old logo).

Done, thanks!

File: index.html [hunk 1]
```diff
@@ -45,7 +45,14 @@
       name="twitter:title"
       content="NestJS - A progressive Node.js framework"
     />
-    <link rel="icon" type="image/x-icon" href="img/favicon.ico" />
```
Comment (kamilmysliwiec) [inline index.html:48]: > <link rel="icon" type="image/x-icon" href="img/favicon.ico" />

Maybe we can keep this one ^ too?

Comment (Arhell) [reply index.html:48]: Done

**Comment by kamilmysliwiec** [comment]
Amazing! Thank you for this contribution @Arhell 🙌

**Pull request merged**